src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SceneExploreServiceFlameGraph.tsx

```diff
@@ -29,6 +34,56 @@ export class SceneExploreServiceFlameGraph extends SceneObjectBase<SceneExploreS
       }),
       body: new SceneFlameGraph(),
     });
+
+    this.addActivationHandler(this.onActivate.bind(this, item));
+  }
+
+  onActivate(item?: GridItemData) {
```

Comment:
When it receives an item (which happens in runtime when a user clicks on a panel action), the Scene object is now responsible for initializing properly the variables it uses.